### PR TITLE
add model address

### PR DESCRIPTION
**Pull request by kuranovandry**

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -6,6 +6,16 @@ def index
     @users = User.all
   end
 
+  def update
+    @user = User.find(params[:id])
```
Comment (maksymst) [inline app/controllers/users_controller.rb]: DRY

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -6,6 +6,16 @@ def index
...
+    if @user.update(user_params)
```
Comment (maksymst) [inline app/controllers/users_controller.rb:11]: why did you use `update` here?

File: db/migrate/20150504122011_create_addresses.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+class CreateAddresses < ActiveRecord::Migration
+  def change
+    create_table :addresses do |t|
+      t.string   :city
+      t.string   :postal_code
```
Comment (maksymst) [inline db/migrate/20150504122011_create_addresses.rb]: where is `limit`?

File: spec/controllers/users_controller_spec.rb [hunk 4]
```diff
@@ -36,6 +36,20 @@
     end
   end
 
+  describe 'PUT #update' do
```
Comment (maksymst) [inline spec/controllers/users_controller_spec.rb:39]: add test for case when user updates existing address. You will see that your functionality does not work

File: spec/factories/address.rb [hunk 5]
```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :address, :class => 'Address' do
+    city { Faker::Company.name }
```
Comment (maksymst) [inline spec/factories/address.rb]: where is relation to `user`?

File: app/controllers/users_controller.rb [hunk 6]
```diff
@@ -6,6 +6,16 @@ def index
...
+      flash[:success] = t('user.successful_update')
+      redirect_to users_path
```
Comment (maksymst) [inline app/controllers/users_controller.rb]: Usually after record updating you need to redirect to record page, but not to the collection page

File: db/migrate/20150504122011_create_addresses.rb (hunk 3 above)
Comment (kuranovandry) [inline db/migrate/20150504122011_create_addresses.rb]: I forget it

File: spec/factories/address.rb (hunk 5 above)
Comment (kuranovandry) [inline spec/factories/address.rb]: I forget it

File: app/controllers/users_controller.rb (hunk 2 above)
Comment (kuranovandry) [inline app/controllers/users_controller.rb:11]: Because I update user and parameters for user

**Pull request merged**